n2y/cache.py

```diff
@@ -0,0 +1,188 @@
+import logging
+import sqlite3
+import pickle
+
+from os.path import exists
+
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+
+class Cache:
+    """Cache Notion block entities in local storage"""
+
+    def __init__(self, client):
```

Comment:
I don't think we should pass `client` as an argument into `cache`, since the `cache` will actually be a child of the `client`. Thus, if we passed the client in it would be a sort of circular reference. Also, I don't think the cache should need to know about the client as best I'm aware.